### PR TITLE
Fav- collection page missing search tiles on fav- page

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-6269

### CODE REVIEW

File: src/tiles/tile-dispatcher.ts [hunk 1]
```diff
@@ -311,6 +312,19 @@ export class TileDispatcher
               @infoButtonPressed=${this.tileInfoButtonPressed}
             >
             </account-tile>`;
+          case 'search':
+            return html`<search-tile
```
Comment (latonv) [inline src/tiles/tile-dispatcher.ts:316]: 👍

File: src/tiles/grid/search-tile.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import { css, CSSResultGroup, html, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { baseTileStyles } from './styles/tile-grid-shared-styles';
+import { BaseTileComponent } from '../base-tile-component';
+import '../image-block';
+
+@customElement('search-tile')
+export class SearchTile extends BaseTileComponent {
```
Comment (latonv) [inline src/tiles/grid/search-tile.ts:8]: Can we add some unit tests for the new component?

**Review by latonv — APPROVED** [review]
LGTM 👍

File: src/tiles/tile-dispatcher.ts [hunk 3]
```diff
@@ -385,6 +399,12 @@ export class TileDispatcher
         --imageBlockBackgroundColor: #f1f1f4;
       }
 
+      search-tile {
+        --tileBorderColor: #555555;
+        --tileBackgroundColor: #666666;
+        --imageBlockBackgroundColor: #666666;
+      }
+
```
[the remark this answers is not in the document]
Comment (nsharma123) [reply src/tiles/tile-dispatcher.ts:407]: Yes, and I am using its own selector `search-tile`

**Pull request merged**